packages/paste-website/src/constants.ts

```diff
@@ -4,6 +4,7 @@ export const PSA_ALERT_HEIGHT = 40;
 export const PASTE_THEME_ALERT_HEIGHT = 54;
 export const SITE_MASTHEAD_HEIGHT = 76;
 export const SITE_BREAKPOINTS = ['768px', '1024px', '1220px'];
+export const SITE_CONTENT_MAX_WIDTH = '1440px';
```

Comment:
After chatting with @corinne-nagel, added this content max width to control the content size within the different slashed background color areas.